internal/transform/scope-css.go

```diff
@@ -85,6 +85,17 @@ outer:
 
 					// if inside @keyframes, don’t transform what’s there
 					if isKeyframes {
+						if strVal == "{" {
+							keyframeCurlyCount++
+						} else if strVal == "}" {
+							keyframeCurlyCount--
+						}
```

Comment:
Previously, we were only doing `out += strVal` inside of this case, but that meant we never updated `keyframeCurlyCount` to reflect when we were done!